docs/advanced-topics/legacy/wf-legacy-registration.rst

```diff
@@ -0,0 +1,88 @@
+Legacy Workflow Registration Methods
+====================================
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. include:: /getting-started/github-apps/note--legacy-dont-sync.rst
+
+If you would prefer not to use GitHub App registration for your workflows, you can instead use one of these methods.
+
+Traditional Registration
+~~~~~~~~~~~~~~~~~~~~~~~~
+When using Bitbucket and GitLab for you workflows, use the traditional registration.
+There are two types of traditional registration: quick registration and manual registration.
+
+There are some ways to make the traditional registration process more seamless.
+
+- For your primary workflow descriptor, use the file suffixes ``cwl``,
+  ``wdl``, ``config`` (for Nextflow), or ``ga`` (for Galaxy) depending on the descriptor language
+  at the root of your repository
+- For your test parameter files, use the file suffix ``json`` at the root
+  of your repository
+- There should be one workflow per repository
+
+By default, Dockstore will search the root of your repository for workflow
+related files. Following the above tips will help streamline the registration
+process, though you can still register workflows with non-standard format by
+using manual registration.
+
+Option A: Quick Register
```

Comment:
I think we want to keep `Quick Register` as the second option in our `getting-started/dockstore-workflows` and `getting-started/dockstore-tools` page. This option is important for more novice users that are not ready to use GitHub.

Reply:
@denis-yuen @coverbeck do you agree or disagree?

Reply:
I think I may be getting confused. This section currently refers to registering using git, are you suggesting replacing or moving it?

Reply:
I think I will let this go and I may be confused by terms. I wanted to suggest we keep an easy user sign-up (non Git-based) in our `getting-started` section for users that are not quite ready for GitHub. I can ask Ash to bring this convo to a future PR so we can simplify this current PR.